googlemaps/client.py

```diff
@@ -133,30 +133,42 @@ def __init__(self, key=None, client_id=None, client_secret=None,
         self.sent_times = collections.deque("", queries_per_second)
 
     def _get(self, url, params, first_request_time=None, retry_counter=0,
-             base_url=_DEFAULT_BASE_URL, accepts_clientid=True, extract_body=None):
+             base_url=_DEFAULT_BASE_URL, accepts_clientid=True,
+             extract_body=None, requests_kwargs=None):
```

Comment:
It's needed to we can tell the requests library to stream on a per-request basis, which is needed for the call to the photos endpoints.